src/auction.js

```diff
@@ -367,8 +367,11 @@ export function newAuction({adUnits, adUnitCodes, callback, cbTimeout, labels, a
   }
 
   function addWinningBid(winningBid) {
+    const { transactionId } = winningBid;
```

Comment:
I don't think this helps; by the time it gets here, it should already be the correct bid. The filtering happens in `markWinningBidAsUsed`.

Reply:
I think for this part I was just aiming to link the winning bid with it's associated adunit to check if deferBilling is true or false.  Looks like that can be done by either comparing `adUnitCode` & `code` between the two or `transactionId`.  Or let me know if there is another way to do this that I may have missed.